### PR TITLE
Responsivenes

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: src/dashboard/events/cd-dashboard-upcoming-event.vue [hunk 1]
```diff
@@ -138,18 +138,21 @@
   @import "../../common/variables";
   @import "../../common/styles/cd-filler-loading";
 
+  @wrap: 300px;
```
Comment (DanielBrierton) [inline src/dashboard/events/cd-dashboard-upcoming-event.vue]: Unused

File: src/dashboard/events/cd-dashboard-upcoming-event.vue [hunk 2]
```diff
@@ -138,18 +138,21 @@
...
   .cd-dashboard-upcoming-event {
     margin: 32px 0;
 
     &__content {
       background: @cd-white;
       display: flex;
       padding: 0 20px;
+      flex-wrap: wrap;
```
Comment (DanielBrierton) [inline src/dashboard/events/cd-dashboard-upcoming-event.vue]: I thought we were leaving flex-wrap out of it now and just letting the breakpoint do it's thing?

Comment (Wardormeur) [reply src/dashboard/events/cd-dashboard-upcoming-event.vue]: Yep, my bad, pre-walk rush :>

File: src/dashboard/events/cd-dashboard-upcoming-event.vue (hunk 1 above)
Comment (Wardormeur) [reply src/dashboard/events/cd-dashboard-upcoming-event.vue]: Indeed :+1:

**Pull request merged**